ndb/src/google/cloud/ndb/__init__.py

```diff
@@ -63,7 +63,6 @@
     "make_connection",
     "MetaModel",
     "Model",
-    "ModelAdapter",
```

Comment:
I would prefer we keep it and replace it by something like `NotImplemented`? If we targeted `>= 3.7` we could use `__getattr__` on the module and just raise an exception on access.

https://www.python.org/dev/peps/pep-0562/

Reply:
Currently we're targeting >= 3.5, it seems. What do our customers require?

Reply:
a8d884a